frontend/package.json

```diff
@@ -4,15 +4,16 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --host",
     "build": "vite build",
     "preview": "vite preview"
   },
   "dependencies": {
+    "@vitejs/plugin-basic-ssl": "^1.1.0",
```

Comment:
generates unsigned certificate that will show a warning but allow the project to run locally and in the local network